File: BOJ/♣ 11000~11999/11536_줄_세우기.js
const isArrayEqual = (arrayA, arrayB) => {
  return arrayA.every((value, index) => value === arrayB[index]);
}

const [, ...input] = require('fs')
  .readFileSync(0, 'utf-8')
  .trim()
  .split('\n');

const increasingArray = [...input].sort();
const decreasingArray = [...increasingArray].reverse();

if (isArrayEqual(input, increasingArray)) {
  console.log('INCREASING');
} else if (isArrayEqual(input, decreasingArray)) {
  console.log('DECREASING');
} else {
  console.log('NEITHER');
}

File: BOJ/♣ 11000~11999/11536_줄_세우기.py
N = int(input())
names = []

for _ in range(N):
    names.append(input())

increasing_names = sorted(names)
decreasing_names = sorted(names, reverse=True)

if names == increasing_names:
    print("INCREASING")
elif names == decreasing_names:
    print("DECREASING")
else:
    print("NEITHER")
